framework/src/application/application.js

```diff
@@ -417,16 +417,11 @@ class Application {
 				'state:updated',
 				'networkEvent',
 				'networkReady',
-				'signature:change',
-				'transactions:change',
+				'newUnconfirmedTransaction',
```

Comment:
What is this event? And since you are refactoring this part, please change names to the pattern of lowercase with scope separated by `:`. So this event will become `transactions:new:unconfirmed`

Reply:
This event is fired when you add the transaction to the pool.
if we gonna follow `:` separated then maybe we should change it to
```
network:event
network:ready
transaction:new
block:new
block:delete
```
?

Reply:
Just one thing, we should stick scope to singular or plural but it would be consistent. e.g. We use `rounds`, so we should follow `blocks` and `transactions`. Or vise versa convert all to singular.